tests/qxmppjingleiq/tst_qxmppjingleiq.cpp

```diff
@@ -1099,5 +1100,97 @@ void tst_QXmppJingleIq::testPayloadTypeRtpFeedbackNegotiation()
     serializePacket(payload2, xml);
 }
 
+void tst_QXmppJingleIq::testRtpErrorCondition_data()
+{
+    QTest::addColumn<QByteArray>("xml");
+    QTest::addColumn<QXmppJingleIq::Reason::RtpErrorCondition>("condition");
+
+    QTest::newRow("NoErrorCondition")
+        << QByteArrayLiteral("<iq type=\"set\">"
+                             "<jingle xmlns=\"urn:xmpp:jingle:1\" action=\"session-terminate\">"
+                             "<reason>"
+                             "<security-error/>"
+                             "</reason>"
+                             "</jingle>"
+                             "</iq>")
+        << QXmppJingleIq::Reason::NoErrorCondition;
+    QTest::newRow("InvalidCrypto")
+        << QByteArrayLiteral("<iq type=\"set\">"
+                             "<jingle xmlns=\"urn:xmpp:jingle:1\" action=\"session-terminate\">"
+                             "<reason>"
+                             "<security-error/>"
+                             "<invalid-crypto xmlns=\"urn:xmpp:jingle:apps:rtp:errors:1\"/>"
+                             "</reason>"
+                             "</jingle>"
+                             "</iq>")
+        << QXmppJingleIq::Reason::InvalidCrypto;
+    QTest::newRow("CryptoRequired")
+        << QByteArrayLiteral("<iq type=\"set\">"
+                             "<jingle xmlns=\"urn:xmpp:jingle:1\" action=\"session-terminate\">"
+                             "<reason>"
+                             "<security-error/>"
+                             "<crypto-required xmlns=\"urn:xmpp:jingle:apps:rtp:errors:1\"/>"
+                             "</reason>"
+                             "</jingle>"
+                             "</iq>")
+        << QXmppJingleIq::Reason::CryptoRequired;
+}
+
+void tst_QXmppJingleIq::testRtpErrorCondition()
+{
+    QFETCH(QByteArray, xml);
+    QFETCH(QXmppJingleIq::Reason::RtpErrorCondition, condition);
+
+    QXmppJingleIq iq1;
+    QCOMPARE(iq1.reason().rtpErrorCondition(), QXmppJingleIq::Reason::NoErrorCondition);
+    parsePacket(iq1, xml);
+
+    const auto rtpErrorCondition1 = iq1.reason().rtpErrorCondition();
+    switch (condition) {
+    case QXmppJingleIq::Reason::NoErrorCondition:
+        QVERIFY(rtpErrorCondition1 == QXmppJingleIq::Reason::NoErrorCondition);
+        break;
+    case QXmppJingleIq::Reason::InvalidCrypto:
+        QVERIFY(rtpErrorCondition1 == QXmppJingleIq::Reason::InvalidCrypto);
+        break;
+    case QXmppJingleIq::Reason::CryptoRequired:
+        QVERIFY(rtpErrorCondition1 == QXmppJingleIq::Reason::CryptoRequired);
```

Comment:
```suggestion
        QVERIFY(rtpErrorCondition1 == QXmppJingleIq::Reason::CryptoRequired);
        break;
```
it's better to also have a break here, so it's not forgotten when adding a new case